apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/block_test.exs

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherRPC.Web.Controller.BlockTest do
+  use ExUnitFixtures
+  use ExUnit.Case, async: false
+  use OMG.WatcherInfo.Fixtures
+
+  import OMG.WatcherInfo.Factory
+
+  alias Support.WatcherHelper
+
+  describe "get_blocks/2" do
+    @tag fixtures: [:phoenix_ecto_sandbox]
+    test "returns the API response with the blocks" do
+      _ = insert(:block, blknum: 1000, hash: <<1>>, eth_height: 1, timestamp: 100)
+      _ = insert(:block, blknum: 2000, hash: <<2>>, eth_height: 2, timestamp: 200)
+
+      request_data = %{"limit" => 200, "page" => 1}
+      response = WatcherHelper.rpc_call("block.all", request_data, 200)
+
+      assert %{
+               "success" => true,
+               "data" => [
+                 %{
+                   "blknum" => 2000,
+                   "eth_height" => 2,
+                   "hash" => "0x02",
+                   "timestamp" => 200
+                 },
+                 %{
+                   "blknum" => 1000,
+                   "eth_height" => 1,
+                   "hash" => "0x01",
+                   "timestamp" => 100
+                 }
+               ],
+               "data_paging" => %{
+                 "limit" => 100,
+                 "page" => 1
+               },
+               "service_name" => "child_chain",
+               "version" => _
+             } = response
+    end
+  end
+end
```

Comment:
I would have added some 'unhappy' path tests, but I know time is short!

Reply:
No you are right. I simply missed this one. Added 23efe6d9342b5f73801fb4ba2002b4cc134f0b98.